### DISCUSSION TITLE
EditableTable.js doesn't allow default values for html type

### DISCUSSION THREAD

**Discussion by bryantAXS** [craftcms/cms#8581] [Ideas]
### Description

The EditableTable macro allows for an `html` type, but the EditableTable.js doesn't include this type inside the switch statement within the createRow method.

In my case, I'm adding a button inside an individual table row, and want that button to be generated when a user adds a new row to the table. For my default value, I'm adding in the html for that button, but it's getting displayed inside a TextField instead.

**Comment by brandonkelly** [comment]
If you want to submit a pull request for this we’d be happy to consider it, but seems like even if this was the case, you’d still need to do some custom JS handling when new rows are created, to add an event listener to the button. So at that point it seems like you might as well just create the button from there as well?
